wdl2cwl/main.py

```diff
@@ -402,7 +405,7 @@ def get_literal_name(
         # if the literal expr is used inside WDL.Expr.Apply
         # the literal value is what's needed
         parent = expr.parent  # type: ignore[union-attr]
-        if isinstance(parent, WDL.Expr.Apply):
+        if isinstance(parent, (WDL.Expr.Apply, WDL.Expr.IfThenElse)):
```

Comment:
values used in either expr Apply or  IfThenElse should return the literal instead of the identity name.